content/en/docs/concepts/cluster-administration/_index.md

```diff
@@ -1,7 +1,71 @@
 ---
-title: "Cluster Administration"
+title: Cluster Administration
+reviewers:
+- davidopp
+- lavalamp
 weight: 100
+content_type: concept
 description: >
   Lower-level detail relevant to creating or administering a Kubernetes cluster.
 ---
 
+<!-- overview -->
+The cluster administration overview is for anyone creating or administering a Kubernetes cluster.
+It assumes some familiarity with core Kubernetes [concepts](/docs/concepts/).
```

Comment:
Is this a direct cut-n-paste from `cluster-administration-overview`?

Reply:
Yep. I used `mv` then tweaked the metadata.